packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/pools/components/delete/controller.js

```diff
@@ -7,11 +7,6 @@ export default class OctaviaLoadBalancerPoolDeleteCtrl {
     this.OctaviaLoadBalancerPoolsService = OctaviaLoadBalancerPoolsService;
   }
 
-  cancel() {
-    this.trackDeleteAction(`cancel`);
-    this.goBack();
-  }
-
```

Comment:
It is better to make the tracking of cancel action here, and remove it from the goBack

Reply:
Done and tested !